build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -70,6 +70,7 @@ output_dir="${output_dir:-${this_dir}/wheelhouse}"
 packages="${packages:-iree-runtime iree-compiler}"
 package_suffix="${package_suffix:-}"
 toolchain_suffix="${toolchain_suffix:-release}"
+enable_cuda="$(uname -m | awk '{print ($1 == "x86_64" || $1 == "aarch64") ? "ON" : "OFF"}')"
```

Comment:
Please add a comment explaining the intent behind this one liner.

Reply:
Oh, and does this match the current behavior, or does it include the changes proposed on https://github.com/iree-org/iree/pull/17564 too? (May want to sequence as two changes)

Reply:
Yes that includes the proposed change. I can move that to a second patch. I would be a PR with only 3-4 lines I think.